addons/project_account/views/project_project_views.xml

```diff
@@ -34,4 +34,15 @@
         <field name="domain">[('account_id', '!=', False)]</field>
     </record>
 
+    <record id="project_embedded_action_analytic_items_dashboard" model="ir.embedded.actions">
+        <field name="parent_res_model">project.project</field>
+        <field name="sequence">105</field>
+        <field name="name">Analytic Items</field>
+        <field name="parent_action_id" ref="project.project_update_all_action"/>
+        <field name="python_method">action_open_analytic_items</field>
+        <field name="context">{"from_embedded_action": true}</field>
+        <field name="groups_ids" eval="[(4, ref('analytic.group_analytic_accounting'))]" />
+        <field name="domain">[('account_id', '!=', False)]</field>
+    </record>
+
```

Comment:
nitpick, but i'd remove that space line.

Reply:
Yeah but file was formatted in way that after odoo tag there is a empty line. I followed the formatting done previously in the file. Have a look at [here](https://github.com/odoo/odoo/blob/53f9f91606a24ba784e90bb8aedd1e249cc0aeb4/addons/project_account/views/project_project_views.xml#L3). Removing the line anyways :smile: